modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -1,70 +1,182 @@
-To learn how to configure storage, see xref:manage:kubernetes/configure-storage.adoc[Configure storage].
+*Requirement*: Before deploying Redpanda to production, each worker node that runs Redpanda must be tuned to optimize the Linux kernel for Redpanda processes. See xref:./kubernetes-tune-workers.adoc[Tuning Kubernetes Worker Nodes for Production].
 
-== Object storage providers for Tiered Storage
+=== Deploy at least three Pod replicas
 
-* Amazon Simple Storage Service (S3)
-* Google Cloud Storage (GCS), using the Google Cloud Platform S3 API
-* Azure Blob Storage (ABS)
+*Recommendation*: Redpanda Data recommends at least three Pod replicas (Redpanda brokers) to use as _seed servers_. Seed servers are used to bootstrap the gossip process for new brokers joining a cluster. When a new broker joins, it connects to the seed servers to find out the topology of the Redpanda cluster. A larger number of seed servers makes consensus more robust and minimizes the chance of unwanted clusters forming when brokers are restarted without any data.
 
-== External networking
+NOTE: By default, the Redpanda Helm chart deploys a StatefulSet with three Redpanda brokers. You can specify the number of Redpanda brokers in the xref:reference:redpanda-helm-spec.adoc#statefulset-replicas[`statefulset.replicas`] configuration.
 
-For external access, each worker node in your cluster must have a static, externally accessible IP address to allow clients to connect to the NodePort Service and access the Redpanda broker.
-Redpanda uses the following default ports:
+=== Set resource requests and limits for memory and CPU
+
+*Recommendation*: In a production cluster, the resources you allocate to Redpanda should be proportionate to your machine type. Redpanda Data recommends that you determine and set these values before deploying the cluster, but you can also update the values on a running cluster. For instructions on setting Pod resources, see xref:manage:kubernetes/manage-resources.adoc[Manage Pod Resources in Kubernetes].
+
+=== Use local PersistentVolumes backed by NVMe disks
+
+*Recommendation*: Use PersistentVolumes that are backed by locally attached NVMe devices to store the Redpanda data directory. NVMe devices outperform traditional SSDs or HDDs.
+
+By default, the Redpanda Helm chart uses the default StorageClass in your Kubernetes cluster to create one PersistentVolumeClaim for each Redpanda broker.
+
+- See xref:manage:kubernetes/storage/volume-types.adoc[]
+- To learn how to configure different volumes see xref:manage:kubernetes/storage/configure-storage.adoc[].
+
+When working with local NVMe disks, provisioning can pose challenges. Dynamic provisioners, though highly scalable and automated, may not always support local PVs. You can either create the PV manually or to automatically create one PV on each node that has local SSDs available, you can use one of the following CSI drivers:
```

Comment:
When working with local NVMe disks, provisioning can pose challenges. Dynamic provisioners, though highly scalable and automated, may not always support local PVs. You can either create the PV manually or to automatically create one PV on each node that has local SSDs available, you can use one of the following CSI drivers:

^^ I'd make these two choices bullets
You can either:

- Create the PV manually
- Automatically create one PV on each node that has local SSDs available using one of the following CSI drivers: